resources/js/processes/screen-builder/components/file-upload-control.js

```diff
@@ -17,28 +17,28 @@ export default {
     inspector: [
       {
         type: "FormInput",
-        field: "label",
+        field: "name",
         config: {
-          label: "Label",
-          helper: "The text to display"
+          label: "Variable Name",
+          name: 'Name',
+          helper: "A variable name is a symbolic name to reference information",
+          validation: 'regex:/^(?:[A-Z_.a-z])(?:[0-9A-Z_. \/a-z])*$/|required'
         }
       },
       {
         type: "FormInput",
-        field: "accept",
+        field: "label",
         config: {
-          label: "File Accepted",
-          helper: "Common file types: application/msword, image/gif, image/jpeg, application/pdf, application/vnd.ms-powerpoint, application/vnd.ms-excel, text/plain"
+          label: "Label",
+          helper: "The label describes the field's name"
         }
       },
       {
         type: "FormInput",
-        field: "name",
+        field: "accept",
         config: {
-          label: "Name",
-          name: 'Name',
-          helper: "The name of the upload",
-          validation: 'regex:/^(?:[A-Z_.a-z])(?:[0-9A-Z_. \/a-z])*$/|required'
```

Comment:
I think we should keep the validation...

Reply:
The validation for "Name" is being kept, we're renaming it to "Variable Name" and moving it to the top to match how the rest of the screen builder configs are laid out.

Reply:
![image](https://user-images.githubusercontent.com/28575916/98407167-09303d80-2024-11eb-920c-99f43a2f8588.png)

Reply:
Great